pkgs/by-name/ti/ticktick/package.nix

```
@@ -14,14 +14,30 @@
   xdg-utils,
   systemd,
 }:
+let
+
+  throwSystem = throw "Unsupported system: ${stdenv.hostPlatform.system}";
+
+  _version = "6.0.21";
+
+  baseUrl = "https://d2atcrkye2ik4e.cloudfront.net/download";
+
+  srcs = {
+    x86_64-linux = fetchurl {
+      url = "${baseUrl}/linux/linux_deb_x64/ticktick-${_version}-amd64.deb";
+      hash = "sha256-e5N20FL2c6XdkDax0SMGigLuatXKZxb9c53sqQ5XVtM=";
+    };
+    aarch64-linux = fetchurl {
+      url = "${baseUrl}/linux/linux_deb_arm64/ticktick-${_version}-arm64.deb";
+      hash = "sha256-6/nzPL+TeEE31S0ngmsUFPZEfWtt4PVAEkMqSa8OpYI=";
+    };
+  };
+in
```

```suggestion
let
  baseUrl = "https://d2atcrkye2ik4e.cloudfront.net/download";
in
```
This was all I meant